setup.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
+import platform
+
+from setuptools import (
+    find_packages,
+    setup,
+)
+
+
+PROJECT_NAME = 'macaroonbakery'
+project = __import__(PROJECT_NAME)
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+requirements = [
+    'requests>=2.2.1'
+]
+
+test_requirements = [
+    'tox',
+    'httmock==1.2.5',
+]
+
+distribution = platform.dist()
+if len(distribution) == 3 and distribution[2] == 'trusty':
+    # Injected into urllib3 to fix insecure Python 2.
+    requirements.extend([
+        'cryptography == 1.3.2',
```

Comment:
Didn't know spaces are allowed around `==` in requirements. Are they?
Anyway, I'd remove them.